File: 5 kyu/ReadableTime.java
public class HumanReadableTime {
	public static String makeReadable(int seconds) {
		
		String hh = Integer.toString((seconds / 3600));
		String mm = Integer.toString((seconds % 3600) / 60);
		String ss = Integer.toString(seconds % 60);
		
		if (hh.length() == 1) {
			hh = "0" + hh;
		}
		if (mm.length() == 1) {
			mm = "0" + mm;
		}
		if (ss.length() == 1) {
			ss = "0" + ss;
		}
		
		return hh + ":" + mm + ":" + ss;
	}
}

//:vault-boy-thumps-up:

File: 5 kyu/ReadableTime.js
function humanReadable(seconds) {

  const minutes = seconds / 60
  const realSeconds = Math.floor(seconds % 60)
  const realMinutes = Math.floor(minutes % 60)
  const realHour = Math.floor(minutes / 60)

  const readableHour =
    String(realHour).length === 1 ? "0" + String(realHour) : realHour

  const readableMinutes =
    String(realMinutes).length === 1
      ? "0" + String(realMinutes)
      : realMinutes

  const readableSeconds =
    String(realSeconds).length === 1
      ? "0" + String(realSeconds)
      : realSeconds

  return `${readableHour}:${readableMinutes}:${readableSeconds}`
}

// wowe

function humanReadable(seconds) {
  var pad = (x) => { return (x < 10) ? "0" + x : x; }
  return pad(parseInt(seconds / (60*60))) + ":" +
         pad(parseInt(seconds / 60 % 60)) + ":" +
         pad(seconds % 60)
}

//:vault-boy-thumps-up:
